Packet++/src/HttpLayer.cpp

```diff
@@ -448,260 +448,76 @@ void HttpRequestFirstLine::setVersion(HttpVersion newVersion)
+    {HttpResponseStatusCode::Http402PaymentRequired, "Payment Required"},
+    {HttpResponseStatusCode::Http403Forbidden, "Forbidden"},
+    {HttpResponseStatusCode::Http404NotFound, "Not Found"},
+    {HttpResponseStatusCode::Http405MethodNotAllowed, "Method Not Allowed"},
+    {HttpResponseStatusCode::Http406NotAcceptable, "Not Acceptable"},
+    {HttpResponseStatusCode::Http407ProxyAuthenticationRequired, "Proxy Authentication Required"},
+    {HttpResponseStatusCode::Http408RequestTimeout, "Request Timeout"},
+    {HttpResponseStatusCode::Http409Conflict, "Conflict"},
+    {HttpResponseStatusCode::Http410Gone, "Gone"},
+    {HttpResponseStatusCode::Http411LengthRequired, "Length Required"},
+    {HttpResponseStatusCode::Http412PreconditionFailed, "Precondition Failed"},
+    {HttpResponseStatusCode::Http413RequestEntityTooLarge, "Request Entity Too Large"},
+    {HttpResponseStatusCode::Http414RequestURITooLong, "Request-URI Too Long"},
+    {HttpResponseStatusCode::Http415UnsupportedMediaType, "Unsupported Media Type"},
+    {HttpResponseStatusCode::Http416RequestedRangeNotSatisfiable, "Requested Range Not Satisfiable"},
+    {HttpResponseStatusCode::Http417ExpectationFailed, "Expectation Failed"},
+    {HttpResponseStatusCode::Http421MisdirectedRequest, "Misdirected Request"},
+    {HttpResponseStatusCode::Http422UnprocessableEntity, "Unprocessable Entity"},
+    {HttpResponseStatusCode::Http423Locked, "Locked"},
+    {HttpResponseStatusCode::Http424FailedDependency, "Failed Dependency"},
+    {HttpResponseStatusCode::Http425TooEarly, "Too Early"},
+    {HttpResponseStatusCode::Http426UpgradeRequired, "Upgrade Required"},
+    {HttpResponseStatusCode::Http428PreconditionRequired, "Precondition Required"},
+    {HttpResponseStatusCode::Http429TooManyRequests, "Too Many Requests"},
+    {HttpResponseStatusCode::Http431RequestHeaderFieldsTooLarge, "Request Header Fields Too Large"},
+    {HttpResponseStatusCode::Http500InternalServerError, "Internal Server Error"},
+    {HttpResponseStatusCode::Http501NotImplemented, "Not Implemented"},
+    {HttpResponseStatusCode::Http502BadGateway, "Bad Gateway"},
+    {HttpResponseStatusCode::Http503ServiceUnavailable, "Service Unavailable"},
+    {HttpResponseStatusCode::Http504GatewayTimeout, "Gateway Timeout"},
+    {HttpResponseStatusCode::Http505HTTPVersionNotSupported, "HTTP Version Not Supported"},
+    {HttpResponseStatusCode::Http506VariantAlsoNegotiates, "Variant Also Negotiates"},
+    {HttpResponseStatusCode::Http507InsufficientStorage, "Insufficient Storage"},
+    {HttpResponseStatusCode::Http508LoopDetected, "Loop Detected"},
+    {HttpResponseStatusCode::Http509BandwidthLimitExceeded, "Bandwidth Limit Exceeded"},
+    {HttpResponseStatusCode::Http510NotExtended, "Not Extended"},
+    {HttpResponseStatusCode::Http511NetworkAuthenticationRequired, "Network Authentication Required"},
+    {HttpResponseStatusCode::HttpStatus1xxCodeUnknown, "1XX Status Code Unknown"},
```

Comment:
added new `HttpStatus1xxCodeUnknown` etc. for better information
code not started from 1xx~5xx should be an error code